### PR TITLE
Update 17 - XML Profile for Signature Validation Tokens.md

### PR DESCRIPTION
**Pull request by Razumain**
Proposed update to XML SVT profile to clarify inclusion of SVT in XML documents

### CODE REVIEW

File: 17 - XML Profile for Signature Validation Tokens.md [hunk 1]
```diff
@@ -102,13 +104,15 @@ When referring to elements defined in this specification
...
 ### 2.1.1. SignatureValidationToken Signature Property
 
 The `<svt:SignatureValidationToken>` element SHALL be placed in a `<ds:SignatureProperty>` element in accordance with \[[XMLDsig](xmldsig)\]. The `<ds:SignatureProperty>` element SHALL be placed inside a `<ds:SignatureProperties>` element inside a `<ds:Object>` element inside a `<ds:Signature>` element.
 
+**Note**: \[[XMLDsig](xmldsig)\] requires the **Target** attribute to be present in `<ds:SignatureProperty>`, referencing the signature targeted by this signature property. If an SVT is added to a signature that do not have an **Id** attribute, implementations SHOULD add an **Id** attribute to the `<ds:Signature>` element and reference that **Id** in the **Target** attribute. This **Id** attribute and **Target** attribute value matching is required by the \[[XMLDsig](xmldsig)\] standard, but it is redundant in the context of SVT validation as the SVT already contains information that uniquely identifies the target signature. Validation application SHOULD not reject an SVT token because of **Id** and **Target** attribute mismatch, and MUST rely on matching against signatue using signed information in the SVT itself.
```
Comment (martin-lindstrom) [inline 17 - XML Profile for Signature Validation Tokens.md:114]: Invalid links. It should say (#xmldsig) 
Are we using boldface for attributes? Use Fixed font instead.
signatue -> signature

File: 17 - XML Profile for Signature Validation Tokens.md [hunk 2]
```diff
@@ -147,6 +151,14 @@ Example:
 </ds:Signature>
 ```
 
+<a name="multiple-svt-tokens"></a>
+### 2.1.2 Multiple SVT in a signature
+If a new SVT is stored in a signature which already contains a previously issued SVT, implementations can choose to either replace the existing SVT or to store the new SVT in addition to the existing SVT.
+
+If the new SVT is stored in addition to the old SVT, then it SHOULD be stored in a new `<ds:SignatureProperty>` element inside the existing `<ds:SignatureProperties>` element where the old SVT is located.
```
Comment (martin-lindstrom) [inline 17 - XML Profile for Signature Validation Tokens.md:158]: remove "then": ... old SVT, it SHOULD be stored ...

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
Use a sensible comment for the commit. That the file was updated is pretty obvious. It will too hard for me to trace down all changes when we are to make a release without good commit comments. The commit comment should also start with "IS-xxx that points at the issue that has been fixed"

Also. Update the date of the spec.

While I'm at it. When creating issues, set a label (for example enhancement) AND add the issue to the current project. Later on you link the PR to the issue and also adds the PR to the project. In that way we get a good traceability between releases.

File: 17 - XML Profile for Signature Validation Tokens.md (hunk 1 above)
Comment (Razumain) [reply 17 - XML Profile for Signature Validation Tokens.md:114]: Updated

File: 17 - XML Profile for Signature Validation Tokens.md (hunk 2 above)
Comment (Razumain) [reply 17 - XML Profile for Signature Validation Tokens.md:158]: Updated

**Pull request merged**